app/services/game-overlay/index.ts

```diff
@@ -228,6 +228,8 @@ export class GameOverlayService extends PersistentStatefulService<GameOverlaySta
   }
 
   async setEnabled(shouldEnable: boolean = true) {
+    if (!this.userService.isLoggedIn()) return;
```

Comment:
I prefer to also have a message in the Settings window, something like `Sing in to use Game Overlays`. Otherwise, it looks broken.